### ISSUE TITLE
[Good First Issue]: Align behavior of ONNX Frontend function ReduceLogSum-11, 13, 18 with original framework

### ISSUE THREAD

**Issue by gkrivor** [openvinotoolkit/openvino#20561]
### Context

Neural networks are graphs consisting of nodes called operators. Each operator corresponds to a mathematical function, usually described in framework's documentation or an AI standard, such as ONNX.
OpenVINO ONNX Frontend is a component responsible for working with ONNX graphs and requires implementation of different ONNX operators in order to use ONNX models.
This task requires alignment between OpenVINO ONNX Frontend and original framework implementations of ReduceLogSum for next list of opsets: [opset 11](https://github.com/onnx/onnx/blob/main/docs/Changelog.md#ReduceLogSum-11),  [opset 13](https://github.com/onnx/onnx/blob/main/docs/Changelog.md#ReduceLogSum-13),  [opset 18](https://github.com/onnx/onnx/blob/main/docs/Changelog.md#ReduceLogSum-18)
Necessary help will be provided by ONNX Fronted team.

### What needs to be done?

First of all, please, take a look on [ReduceMax](https://github.com/openvinotoolkit/openvino/pull/23475) PR for a reference.

Operator details can be found in [ONNX Operators](https://github.com/onnx/onnx/blob/main/docs/Operators.md#ReduceLogSum)
More details can be found in ONNX Changelog: [opset 11](https://github.com/onnx/onnx/blob/main/docs/Changelog.md#ReduceLogSum-11),  [opset 13](https://github.com/onnx/onnx/blob/main/docs/Changelog.md#ReduceLogSum-13),  [opset 18](https://github.com/onnx/onnx/blob/main/docs/Changelog.md#ReduceLogSum-18)
1. Function already has a common implementation in OpenVINO. First of all, you need to review a documentation and prepare a table with differences between versions. It could be, for instance, a missing property, extended/reduced coverage of existing property, etc...
2. Copy existing implementation [here](https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/onnx/frontend/src/op) to make it aligned with original framework (extend or reduce coverage of a common implementation). Copy of modified implementation should be in a defined opset, or in opset 1 in case it implements oldest implementation. [Example of multi-opset operation](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/src/op/softmax.cpp#L50).
3. Register the function in [ops_bridge.cpp](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/src/ops_bridge.cpp#L554-L560) while keeping alphabetical order
4. Create test model(s) in [ONNX models directory](https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/onnx/tests/models/). OpenVINO test infrastructure then converts prototxt files to ONNX models - you will use those models later in tests
5. Add tests covering all use cases [here](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/tests/onnx_import.in.cpp)
6. Check [Python xfailed tests](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/tests/tests_python/test_backend.py) to find a test marked as a xfailed for added functionality. If any exist - remove corresponding lines and try to verify by using cmdline "python -m pytest -k name_of_test".
More details in [adding operators to ONNX Frontend guide](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/docs/how_to_add_op.md)

### Resources

- [Contribution guide - start here!](https://github.com/openvinotoolkit/openvino/blob/master/CONTRIBUTING.md)
- [Intel DevHub Discord channel](https://discord.gg/wPuqAujS) - engage in discussions, ask questions and talk to OpenVINO developers
- [What is OpenVINO?](https://github.com/openvinotoolkit/openvino#what-is-openvino-toolkit)
- [User documentation](https://docs.openvino.ai/)
- Guides on [ONNX Frontend architecture](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/docs/architecture.md) and [ONNX Frontend tests](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/docs/tests.md) are also available
- You can also see a [blog post on contributing to OpenVINO](https://medium.com/openvino-toolkit/how-to-contribute-to-an-ai-open-source-project-c741f48e009e)

### Contact points

@gkrivor

**Comment by Hunter-bitbit** [comment]
Hi! I'm a first time contributor. Could this issue be assigned to me?

**Comment by mlukasze** [comment]
It's yours now :)

**Comment by Hunter-bitbit** [comment]
@mlukasze I'm wondering if it's realistic for me to complete this assignment without any real knowledge or experience in AI. I've started on the assignment and I very much want to contribute but it is a lot to take in. Any tips or suggestions? Should I talk with anyone in particular on the Intel DevHub Discord?

**Comment by p-wysocki** [comment]
Hello @Hunter-bitbit, do you need any help from us? :)

**Comment by Hunter-bitbit** [comment]
Hi @p-wysocki, some help would be great, yes. I'm mostly just having issues with understanding what needs to be done. I made the table of differences between versions, but steps 2 and 3 don't make much sense to me. I found the existing implementation but I don't know where to copy it to? For the registration I found a ReducedLogSum in there... do I have to specify the namespace it's in? I'm just very confused as to what I actually _have_ to do since it seems like everything is already implemented... sorry for the hassle :\

**Comment by p-wysocki** [comment]
Hello @Hunter-bitbit, it may be the case that the operator is currently correctly aligned. Could you please give a brief summary of the changes between op versions and what OpenVINO has? That would simplify the discussion a lot. cc @gkrivor

**Comment by mengbingrock** [comment]
.take

**Comment by mengbingrock** [comment]
Hi, 
The PR is created:
[23522](https://github.com/openvinotoolkit/openvino/pull/23522)
